### ISSUE TITLE
Have a look of Organ sample/model generated smiles file!

### ISSUE THREAD

**Issue by toushi68** [molecularsets/moses#21]
This is what I did:
(moses) [trial6@xps8500 moses]$ python scripts/run.py --gpu 0 --model organ
the metrics generated looks OK.
(moses) [trial6@xps8500 moses]$ more metrics.csv 
FCD/Test,FCD/TestSF,Filters,Frag/Test,Frag/TestSF,IntDiv,IntDiv2,NP,QED,SA,SNN/Test,SNN/TestSF,Scaf/Test,Scaf/TestSF,logP,model,unique@1000,unique@10
000,valid,weight
47.03170334364395,48.812974148127694,0.9452942921310225,0.7442927316911858,0.7363127970096939,0.6648230388481462,0.6524179217292552,1.860596282806586
,0.5782202256571298,2.5691228002718356,0.37832045248554563,0.329009914928975,0.5812598568222165,0.011773956277315145,139.47112549646172,organ,0.981,0
.9175,0.7866333333333333,524984.1029653098
However, if you visually check the generated smiles:
(moses) [trial6@xps8500 moses]$ head data/organ_generated.csv 
SMILES
CCOC(=O)CNC(=O)C(C)NCCCCC(=O)NCCCCC(=O)NCCCCC(=O)NCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
COCCNC(=O)CNC(=O)C(C)N(C)C(=O)C(C)N(C)C(=O)C(C)NC(=O)C(C)N(C)C(=O)C(C)N(C)C(=O)c1ccccc1NC(=O)C(C)OCC
CC(C)C(=O)Nc1ccccc1NC(=O)C(C)N(C)C(=O)C(C)N(C)C(=O)C(C)N(C)C(=O)C(C)N(C)C(=O)C(C)N(C)C(=O)C(C)N(C)C(
CCOCCCCNC(=O)C(C)NC(=O)C(C)NC(=O)C(C)N(C)C(=O)C(C)N(C)C(=O)C(C)NCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
O=C(CCCO)NCCNC(=O)COC(=O)C(C)OC(=O)CCCCC(=O)NCCCCC(=O)NCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
Cc1ccccc1NC(=O)C(C)C(=O)NCCCCC(=O)NCCCCC(=O)NCCCCCC(=O)NCCCC(=O)NCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
CC(C)NC(=O)C(NC(=O)C(C)OCCNC(=O)c1ccccc1)C(=O)NCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCNC(=O)C(
COCCOC(=O)CCCCCCCCCCCCCNC(=O)C(C)C(=O)NCCCCCC(=O)NCCCCCC(=O)NCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
CC(C)CCNC(=O)C(C)NC(=O)C(C)N(C)C(=O)C(C)N(C)C(=O)C(C)NC(=O)C(C)N(C)C(=O)C(C)NC(=O)C(C)N(C)C(=O)C(C)N
Any idea?
btw, aae/char_rnn generated smiles look much more reasonable.

**Comment by toushi68** [comment]
True, using QED can avoid this type of novel molecules. But do you have any data to support the advantage of using FrechetMetrics over direct calling QED? Calling FrechetMetrics is very slow and  the GPU usage will be easily lost after certain pg optimizations and turned back to CPU calculation.
The worst case is to call FCD metrics as the reward function. FCD metrics calls Keras/tensorflow several times for one pg iteration. Hope someone can rewrite it with pytorch and then we can see the advantage of FCDMetrics (as a reward function).
